build/release-tars/BUILD

```diff
@@ -69,7 +69,6 @@ NODE_TARGETS = [
 # since we include them in the actual build rule.
 SERVER_TARGETS = [
     "//cmd/hyperkube",
-    "//cmd/kube-aggregator",
```

Comment:
This can change to '//vendor:k8s.io/kube-aggregator_bin' which should be the same.